src/main.ts

```diff
@@ -139,7 +141,6 @@ export function activate(context: vscode.ExtensionContext): void {
         new CustomViewsFeature(),
         new FoldingFeature(logger, documentSelector),
     ];
-
```

Comment:
What's the point of getting rid of this blank line?

Reply:
Because reasons?

Honestly, not sure. Must have been by accident.